sdk/test-utils/recorder-new/src/utils/relativePathCalculator.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import fs from "fs";
+import { RecorderError } from "./utils";
+
+export function relativeRecordingsPathForNode(testContext: Mocha.Test) {
+  const testAbsolutePath = testContext.file;
+  if (!testAbsolutePath) {
+    throw new RecorderError("Unable to grab the file path from the test run");
+  }
+
+  const recordingsPath = findRecordingsFolderPath(testAbsolutePath);
```

Comment:
All this can be replaced with something like `testAbsolutePath.split(path.sep)` and then looking through the resulting array by index.

Reply:
updated